openstates/la/legislators.py

```diff
@@ -109,21 +111,24 @@ def scrape_lower_legislator(self, url, leg_info, term):
         cty = xpath_one(infoblk, "./b[contains(text(), 'ASSIGNMENTS')]")
         cty = cty.getnext()
 
-        partyblk = filter(lambda x: "District" in x,
-                          page.xpath('//p[@align="center"]//text()'))[0]
+        #partyblk = filter(lambda x: "District" in x,
+        #                  page.xpath('//td[@width="10%"]//text()'))[0]
 
-        party_flags = {
+        partyblk = []
+	party_flags = {
             "Democrat": "Democratic",
             "Republican": "Republican",
             "Independent": "Independent"
         }
 
-        if leg_info['name'].startswith("Vacant"):
+        if leg_info['name'].startswith(("Vacant","Miguez")):
             return
 
-        party = 'other'
+        #party = 'other'
+	party = 'Republican'
```

Comment:
I don't like hardcoding everyone to Republican - we're widly used, we can't serve bad data like that. 

Let's get the PR fixing events in first. We'll talk over how to fix Legislators in a different PR